changelog/unreleased/3137

```diff
@@ -0,0 +1,9 @@
+Bugfix: Sorted collaborators column, deduplicate public entry
+
+The collaborators appears a the column in the "shared with others"
+section are now sorted: first by share type (user, group, remote, link) and then by
+display name using natural sort.
+Additionally, if there is more than one public link for the resource, the text "Public"
+only appears once in the collaborators column.
+
+https://github.com/owncloud/phoenix/issues/3137
```

Comment:
Add PR pls

Reply:
done. it was the usual chicken-egg thing